disco/bot/bot.py

```diff
@@ -312,6 +312,7 @@ def get_commands_for_message(self, require_mention, mention_rules, prefix, msg):
                         if member.nick:
```

Comment:
This line and the following line can now be deleted with your change, these where the previous check but I don't see any value in only handling a nickname-specific mention if the member has a nickname.

Reply:
That makes sense. I pushed another commit now.